examples/api/test/material/bottom_app_bar/bottom_app_bar.2_test.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/bottom_app_bar/bottom_app_bar.2.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Floating Action Button visibility can be toggled',
+      (WidgetTester tester) async {
```

Comment:
My point about long test descriptions what about those lines starting with `testWidgets`, you can either format them on a single line such as:

```dart
    testWidgets("description", (WidgetTester tester) async {
      ...
    });

```

or on multiple lines if the description is very long, using the following format:

```dart
    testWidgets(
      "very long description",
      (WidgetTester tester) async {
        ...
      },
    );

```

Reply:
> @bleroux single quotes and I think you meant "was" and not "what"

Good catch :pray: , I updated the comment, especially the code part because it might serve as a reference.

Reply:
@bleroux From what I understand, it's currently the way it is, right? Even when I run dart format, it doesn't change, at least not for the line you mentioned.

Reply:
@IldySilva We are not suppose to use dart format on the Flutter repo (as it often results on non readable code, there are ongoing work on the dart formatter that will solve this). Nonetheless for some tests it could be ok as long as you just apply it to code section (if applied to a whole test file, it will change a lot of existing lines).

In the case I pointed, dart format will give you the right result if you add a comma after the `testWidget` second parameter.

```dart
    testWidgets(
      'Very long description',
      (WidgetTester tester) async {
        ...
      }, // This comma is important to get the right result with dart format
    );

```